src/ApiResource/Error.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\ApiResource;
+
+use ApiPlatform\Exception\ErrorExceptionInterface;
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\NotExposed;
+
+#[NotExposed(uriTemplate: '/errors/{statusCode}')]
+class Error
```

Comment:
```suggestion
final class Error
```

Reply:
I was hesitant about this, should we (or not) allow users to extend it for their custom Errors ?

Reply:
not the resource itself no